src/screens/Tools/Tools.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { Fragment } from 'react';
+import AppMetaTags from 'components/AppMetaTags/AppMetaTags';
+import PageContent from 'components/PageContent';
+import { Heading2 } from 'components/Markdown';
+import { formatMetatagDate } from 'common/utils';
+import { LearnHeading1 } from '../Learn/Learn.style';
+import {
+  productsLandingContent,
+  ProductsLandingSection,
+} from 'cms-content/learn';
+import {
+  MarkdownTools,
+  ToolsSection,
+  EmbedExampleWrapper,
+} from './Tools.style';
+import { TocItem } from 'cms-content/utils';
+
+/**
+ * When updating the COVID_RESPONSE_SIMULATOR_URL, update the version number
+ * and the URL in src/assets/covid-response-simulator-version.txt. This file is
+ * used by the the CRS to determine if a copy is up to date, and show a
+ * message to the user.
+ */
+export const COVID_RESPONSE_SIMULATOR_URL =
+  'https://docs.google.com/spreadsheets/u/4/d/1vs-OjzBuE96sky-8CNJWLPWnXkJwsDKw5XBejYR5-uk/copy?usp=sharing';
+
+const {
+  header,
+  productsList,
+  metadataTitle,
+  metadataDescription,
+} = productsLandingContent;
+
+export const sidebarSections: TocItem[] = [
+  {
+    label: 'Tools',
+    to: '/tools',
+    items: productsLandingContent.productsList.map(product => ({
+      to: `/tools#${product.productId}`,
+      label: product.productName,
+    })),
+  },
+];
```

Comment:
nit: Consider moving this to the CMS as we do for some Learn pages (not strong feelings about this, but I think it makes sense to centralize document structure there and just use the structure and content in screens).

Reply:
per our chat, punting this to a larger consideration of where these URLs should live (along with learn TOC urls, breadcrumbs, etc)